### ISSUE TITLE
how to cite SKOPE?

### ISSUE THREAD

**Issue by kbvernon** [openskope/skopeui#202]
Sorry if this exists already (I looked but couldn't find it), but it would be nice if the website included a "How to cite SKOPE" page, including a bibtex formatted citation that someone could just copy/paste into their reference manager.

I commonly use Bocinsky 2014/2016 for paleoCAR (I also cite the R package), but don't have a general reference for the whole SKOPE app. I think having that would also make it easier for you all to track how often the app is getting used for research, among other things.

Thanks!

**Comment by alee** [comment]
There's an example citation in the Terms of Use but perhaps we should make it more prominent?

![image](https://github.com/openskope/skopeui/assets/22534/dc48f9a2-44fe-41e3-ad27-f5cd146ef993)

**Comment by bocinsky** [comment]
Yeah, thanks Blake. I think perhaps we could add it automatically to the info sheet for each dataset, below where we have the References for the dataset.

**Comment by kbvernon** [comment]
Ah, yeah, I see it now. You have to go into the app to find it. In addition to what Kyle said, I imagine researchers would appreciate making that easier to find. Maybe include it under the `Use the SKOPE App` tab on the main page?

**Comment by kbvernon** [comment]
It's not mission critical or anything, but maybe consider including the bibtex formatted references, too? Or even a .bib file with all the references!

```
@misc{
    skope2022,
    type = {Web Application},
    title = {Synthesizing {Knowledge} of {Past} {Environments} ({SKOPE})},	
    url = {https://app.openskope.org/},
    language = {en},	
    author = {Bocinsky, K and Gillreath-Brown, A and Kintigh, K and Kinzig, A and Kohler, T and Lee, A and Ludascher, B and McPhillips, T and Nguyen, C and Pritchard, C},
    doi = {10.5281/zenodo.6522974}
}
```

**Comment by alee** [comment]
can do, the citation that's listed is generated from our citation.cff file so I'll update that and see about providing a .bib file as well.

https://github.com/openskope/skopeui/blob/main/CITATION.cff

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files

**Comment by alee** [comment]
I've added a `Cite SKOPE` button to the footer that pops up a modal with a SKOPE citation generated from our citation.cff  file. You can click on the citation itself to copy it to your clipboard.

![image](https://github.com/openskope/skopeui/assets/22534/cec876c3-3708-43fc-9747-744ae120462e)

Please test it out on the bottom of the page at https://staging.openskope.org and let me know if you find any strangeness. I'll deploy to prod next week if it seems OK.

**Comment by kbvernon** [comment]
Looks great! Thanks! 

Do you want to wait till next week to close this issue yourself? Or would you prefer that I go ahead and close it now?

**Comment by alee** [comment]
I'm OK with leaving this open until I add the citation to each dataset's metadata sheet as well under "References" (there's also some work needed in this client-side app to resolve https://github.com/openskope/skope-api/issues/44)
